public/js/components/App.js

```diff
@@ -35,17 +35,26 @@ const App = React.createClass({
   componentDidMount() {
     this.shortcuts = new KeyShortcuts({ window });
     this.shortcuts.on("Cmd+P", this.toggleSourcesSearch);
+    window.addEventListener("keydown", this.onKeyDown);
```

Comment:
oh interesting - i found the library fairly readable at key-shortcuts, might be worth a quick read.